.github/workflows/build.yml

```
@@ -0,0 +1,175 @@
+
+    strategy:
+      matrix:
+        service: [ "${{ github.event.inputs.service }}" ]
+
+    outputs:
+      docker_image: ${{ steps.save_docker_image.outputs.docker_image }}
+      db_image:     ${{ steps.save_db_image.outputs.db_image }}
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)